packages/storage-postgres/src/schema/space.ts

```diff
@@ -15,8 +15,20 @@ export const spaces = pgTable('spaces', {
   categories: jsonb('categories')
     .$type<Array<(typeof categories.enumValues)[number]>>()
     .default([]),
+  parentId: integer('parent_id'),
   ...commonDateFields,
```

Comment:
Let's declare the reference 🙏
```suggestion
  parentId: integer('parent_id').references((): AnyPgColumn => spaces.id),
```